index.html

```diff
@@ -1368,8 +1371,8 @@ <h2>Verification Methods</h2>
       </p>
 
       <p>
-In order to maximize global interoperability, the verification method properties
-SHOULD be registered in the [[DID-SPEC-REGISTRIES]].
+In order to maximize global interoperability, the verification method property
+names SHOULD be registered in the [[DID-SPEC-REGISTRIES]].
```

Comment:
```suggestion
In order to maximize global interoperability, the verification method 
SHOULD be registered in the [[DID-SPEC-REGISTRIES]].
```

Reply:
Technically, we're registering more than just the property name, so keeping this less specific for now.